ArduCopter/Copter.h

```diff
@@ -473,6 +473,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     Location_Class current_loc;
     Location ekf_origin;
     Vector3f current_pos;
+    Vector3f current_vel;
```

Comment:
Re adding current_vel, one small thing is that it adds an additional dependency from the mode code to the main vehicle code which makes it harder to separate the mode code from the vehicle.